ci/taskcat.yml

```diff
@@ -34,6 +34,12 @@ tests:
       EnvironmentName: xldevops-test
       LoadBalancerType: internet-facing
       InstallBastionHost: 'false'
+      # If you have a BASE64 license then use below AcceptEula, XLDeployLicense and XLReleaseLicense parameter values
+      AcceptEula: none
       XLDeployLicense: BASE64
       XLReleaseLicense: BASE64
+      # If you do not have a BASE64 license then use below AcceptEula, XLDeployLicense and XLReleaseLicense parameter values
+      # AcceptEula: yes
+      # XLDeployLicense: trial
+      # XLReleaseLicense: trial
```

Comment:
Why are you committing a commented block?

Reply:
This block is used only if the user does not have a BASE64 license. Please refer to scenario #2 

**Quick start if you do not have a license**: (use below AcceptEula, XLDeployLicense, and XLReleaseLicense parameter values in [taskcat.yml](https://github.com/xebialabs/quickstart-xebialabs-devops-platform/blob/LOVE-1577/ci/taskcat.yml) file)

 AcceptEula: yes
 XLDeployLicense: trial
 XLReleaseLicense: trial

If the user has BASE64 license then use scenario #1

**Quick start if you have a license**: (use below AcceptEula, XLDeployLicense and XLReleaseLicense parameter values in [taskcat.yml](https://github.com/xebialabs/quickstart-xebialabs-devops-platform/blob/LOVE-1577/ci/taskcat.yml) file)

  AcceptEula: none
  XLDeployLicense: BASE64
  XLReleaseLicense: BASE64

I have commented that block just for either or condition of license availability

Reply:
Please hold on PR review as email communication is still going on for EULA approach in AWS Quickstart. Once it is finalized then I'll commit corrected files and let you know for the review.